### PR TITLE
Rating component added

### PR DESCRIPTION
**Pull request by Raghav888**
1. Created rating component
2.  Updated getting started data
3. Updated sidebar

### CODE REVIEW

**Review by neetukumari4858 — APPROVED** [review]

**Review by Omkar-Chaskar — APPROVED** [review]
Looks Good

File: component/component.css [hunk 1]
```diff
@@ -90,8 +92,8 @@
     --height-img: 100%;
     --card-vert-width-max: 14rem;
     --margin-card-badge: 2rem 0px 0px 0.7rem;
-    --text-overlay-top: 50%;
-    --text-overlay-left: 30%;
+    --text-overlay-top: 45%;
```
Comment (shashidwivedi) [inline component/component.css:95]: You have two different spellings for text in your variable naming convention, it is good to use only one, either 'text' or 'txt'
'text' is more preferable though

Comment (Raghav888) [reply component/component.css:95]: Yes will use only text. In next PR will change it

**Review by paarthbajaj — COMMENTED** [review]
Everything looks good in my opinion.

**Pull request merged**